### PR TITLE
Class based prop tests up to float

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: property_tests/tests/class_syntax/blob/dfx.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        "canister": {
+            "type": "azle",
+            "main": "src/index.ts",
+            "candid_gen": "automatic",
+            "declarations": {
+                "output": "test/dfx_generated/canister",
+                "node_compatibility": true
+            },
+            "custom": {
+                "experimental": true
```
Comment (lastmjs) [inline property_tests/tests/class_syntax/blob/dfx.json:12]: Experimental must be removed from all class-based examples if possible

File: property_tests/tests/class_syntax/blob/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "scripts": {
+        "test": "tsx test/test.ts"
+    },
+    "dependencies": {
+        "azle": "0.23.0"
+    },
+    "devDependencies": {
+        "tsx": "^4.15.7",
+        "typescript": "^5.2.2"
```
Comment (lastmjs) [inline property_tests/tests/class_syntax/blob/package.json:10]: Can you get rid of typescript and see if that works? If it works, can you remove it from all property tests?

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: property_tests/tests/class_syntax/blob/dfx.json (hunk 1 above)
Comment (bdemann) [reply property_tests/tests/class_syntax/blob/dfx.json:12]: Done

File: property_tests/tests/class_syntax/blob/package.json (hunk 2 above)
Comment (bdemann) [reply property_tests/tests/class_syntax/blob/package.json:10]: done

**Pull request closed without merging**